pom.xml

```diff
@@ -53,7 +72,6 @@
                     <version>3.8.0</version>
                     <configuration>
                         <release>13</release>
-                        <compilerArgs>--enable-preview</compilerArgs>
```

Comment:
I'm assuming this will break things in the future since the preview is required for the actual 13 features

Reply:
JavaFX won't run when we use --enable-preview.